### PR TITLE
Created Unit Test and bugg fixes on Customer section

### PR DESCRIPTION
**Pull request by andreholmlund**

### CODE REVIEW

File: BankConsole/Program.cs [hunk 1]
```diff
@@ -176,35 +175,83 @@ public static void DisplayMenu(DatabaseRepo _repo, BankLogic bankLogic)
...
-                var orgNo = Console.ReadLine();
+                while (!int.TryParse(Console.ReadLine(), out orgNo))
+                {
+                    Console.WriteLine("Please Enter a valid numerical value!");
+                    Console.WriteLine("Please Enter an ID number:");
+                }
+
                 Console.WriteLine("Adress:");
                 var adress = Console.ReadLine();
+                while (String.IsNullOrEmpty(name))
```
Comment (CharbelIlias) [inline BankConsole/Program.cs]: name?

**Review by CharbelIlias — COMMENTED** [review]
Felaktig variabel

File: BankConsole/Program.cs (hunk 1 above)
Comment (andreholmlund) [reply BankConsole/Program.cs]: Okej, fixar!

File: BankConsole/Program.cs [hunk 2]
```diff
@@ -198,7 +198,7 @@ public static void DisplayMenu(DatabaseRepo _repo, BankLogic bankLogic)
 
                 Console.WriteLine("Adress:");
                 var adress = Console.ReadLine();
-                while (String.IsNullOrEmpty(name))
```
Comment (CharbelIlias) [inline BankConsole/Program.cs]: Ser bra ut.

**Pull request merged**